docs/DeveloperGuide.md

```diff
@@ -232,17 +232,18 @@ The activity diagram shows the general sequence of steps when a user interacts w
 
 ### Edit person feature
 
-#### Implementation
-
 #### Design considerations
+**Aspect: Check if the person with person id exists before editing**
```

Comment:
Minor change, but perhaps can use the code-formatted `personId` to be consistent with the code